src/main/clojure/clara/rules/durability/fressian.clj

```diff
@@ -132,7 +132,11 @@
   ([^Reader rdr]
    (read-record rdr nil))
   ([^Reader rdr add-fn]
-   (let [builder (-> (.readObject rdr) resolve deref)
+   (let [try-resolve #(if-let [resolved (resolve %)]
+                        resolved
+                        (throw (ex-info (str "Unable to resolve fact type symbol: '" % "'")
```

Comment:
I don't think this is necessarily a fact, although if the Fressian handlers are being brought in by an IWorkingMemorySerializer implementation they usually will be - the Clara engine nodes have their own handlers anyway: https://github.com/cerner/clara-rules/blob/0.21.2/src/main/clojure/clara/rules/durability/fressian.clj#L393 So this seems OK either way as "fact type symbol" or "record constructor symbol".